chunky/src/java/se/llbit/chunky/renderer/renderdump/DumpFormat.java

```diff
@@ -72,12 +72,16 @@ protected void readSamples(DataInputStream inputStream, Scene scene, IntConsumer
 
   public void merge(DataInputStream inputStream, Scene scene, TaskTracker taskTracker)
       throws IOException, IllegalStateException {
-    try (TaskTracker.Task task = taskTracker.task("Merging render dump", scene.canvasWidth() * scene.canvasHeight())) {
+    try (TaskTracker.Task task = taskTracker.task("Merging render dump", scene.canvasHeight())) {
+      int width = scene.canvasWidth();
       int previousSpp = scene.spp;
       long previousRenderTime = scene.renderTime;
 
       readHeader(inputStream, scene);
-      mergeSamples(inputStream, previousSpp, scene, task::update);
+      mergeSamples(inputStream, previousSpp, scene, prog -> {
+        if (prog % width == 0)
+          task.update(width);
```

Comment:
```suggestion
          task.update(prog);
```

Reply:
Nice catch. Fixed in f11c3c6. It should be `prog/width` since this is tracking line by line and not pixel by pixel.